### DISCUSSION TITLE
Crashing during sync?

### DISCUSSION THREAD

**Discussion by radxrico** [Taxel/PlexTraktSync#953] [Support] [answered]
Hello,

Fantastic script! I just have an issue where it keeps crashing at random times. Normally I just keep re-opening it and it eventually gets through it. However on the current category "Documentaries" on my movie server. It gets to 95% done then it crashes. It's been like 10+ times and I don't know how to edit the categories to maybe skip that one to get on with the rest of my syncing. 

Any suggestions would be awesome! Pretty new at this :)

EDIT: I have the windows setup install and just run the shortcut every time if that helps.

**Comment by glensc** [comment, marked as answer]
Use new issue to report a crash, be sure to include crash exception in it

**Reply by radxrico** [reply]
Thank you for the response glensc. Forgive my stupidity I have not done this before, but where can I fin the crash exception?

**Reply by glensc** [reply]
You said it crashes:
> It gets to 95% done then it crashes

so, copy that terminal output with the stack trace.
